jim/markov.py

```diff
@@ -24,26 +23,24 @@ def chains(self):
                 if symbol is None:
                     break
                 chain.append(symbol)
-            yield chain
 
+            yield chain
 
     def _get_starting_symbol(self):
         if self._starting_symbols:
             starting_symbol = random.choice(self._starting_symbols)
+
             return random.choice(self._memory[starting_symbol])
         else:
-            return random.choice(list(self._memory.keys()))            
-
+            return random.choice(list(self._memory.keys()))
```

Comment:
Style: this particular return was fine. I only put in comments where I had issues with it.

Reply:
I'm using a new text editor and accidentally left white space at the end of the line in the previous commit